source/install/self-managed-changelog.md

```diff
@@ -22,7 +22,10 @@ Mattermost v7.9.0 contains a low severity level security fix. [Upgrading](https:
 
 ### Important Upgrade Notes
 
- - Added a new index on ``Posts(OriginalId)``. For a database with 11.8 million posts, on a machine with a i7-11800H CPU (8 cores, 16 threads), 32GiB of RAM and SSD, the index creation takes 98.51s on MYSQL and 2.6s on PostgreSQL. To avoid any table locking, servers using PostgreSQL are recommended to create the index online before performing the upgrade using the ``CONCURRENTLY`` option, as in ``CREATE INDEX CONCURRENTLY idx_posts_original_id ON Posts(OriginalId);``.
+- Added a new index on ``Posts(OriginalId)``. For a database with 11.8 million posts, on a machine with a i7-11800H CPU (8 cores, 16 threads), 32GiB of RAM and SSD, the index creation takes 98.51s on MYSQL and 2.6s on PostgreSQL. 
+- In PostgreSQL databases, the ``Posts`` table will be locked during index creation. To avoid locking this table, admins can create the index manually before performing the upgrade using the following non-blocking query: ``CREATE INDEX CONCURRENTLY idx_posts_original_id ON Posts(OriginalId);``. 
+- Admins managing PostgreSQL deployments containing fewer posts may prefer that the migration process create the index, and accept that ``Posts`` table will remain locked until the migration is complete.
```

Comment:
I'm very scared of suggesting a grammar fix to a technical writer, but here I go. I'll be happy to stand corrected and learn some new grammar if this is wrong.
```suggestion
- Admins managing PostgreSQL deployments containing fewer posts may prefer that the migration process creates the index, and accept that ``Posts`` table will remain locked until the migration is complete.
```